src/main/java/seedu/contax/logic/commands/EditPriorityCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.contax.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.contax.commons.core.Messages;
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.exceptions.CommandException;
+import seedu.contax.model.Model;
+import seedu.contax.model.appointment.Appointment;
+import seedu.contax.model.appointment.Priority;
+
+
+/**
+ * Edits the details of an existing priority in the address book.
+ */
+public class EditPriorityCommand extends Command {
+    public static final String COMMAND_WORD = "editpriority";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the level of priority.\n"
```

Comment:
This is a pretty vague message, priority of what?

Reply:
Thx will change description

Reply:
I dont see a change here.

Reply:
thx edited